src/pages/FolderListPage/index.js

```diff
@@ -0,0 +1,109 @@
+import "./style.css";
+import { useEffect, useState } from "react";
+import Nav from "../../components/Nav";
+import Footer from "../../components/Footer";
+import AddLinkBar from "../../components/AddLinkBar";
+import SearchBar from "../../components/SearchBar";
+import FilterButton from "../../components/FilterButton";
+import Folders from "../../components/FolderItem";
+import { getFolderLinks, getTotalFolderLinks } from "../../api";
+
+import addIcon from "../../assets/add.svg";
+import shareIcon from "../../assets/share.svg";
+import penIcon from "../../assets/pen.svg";
+import deleteIcon from "../../assets/delete.svg";
+
+const FolderPage = ({ user, folderObj }) => {
+  const [keyword, setKeyword] = useState("전체");
+  const [links, setLinks] = useState([]);
+
+  const handleLoad = async () => {
+    const links = await getTotalFolderLinks();
+    if (links.data.length) {
+      setLinks(links.data);
+    }
```

Comment:
```suggestion
    if (links.data) {
      setLinks(links.data);
    }
```
[중요도: 높음] 가 더 맞는 표현으로 보입니다!
만약 links.data가 배열이 아닐 경우에는 에러가 발생할 수 있는 요소로 보입니다.

Reply:
피드백 감사합니다! https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/217/commits/207e5e8dbf37752118c32e9ceffa08e517dbfda0